cengage/react-magma
# fix(Input): Clarification that developers should use `ref` with `Input`. Add a new example for Storybook.

Issue: #1124

## What I did

Add clarification that developers should use `ref` with `Input`. Add a new example for Storybook.

## Screenshots:


## Checklist 

## How to test

Open `Input` in the docs -> Open `Error message` -> Leave the input blank and click `Submit` -> Check that the input has focus.
